### ISSUE TITLE
Options.Items not read aloud by ScreenReader

### ISSUE THREAD

**Issue by jason-instructure** [instructure/instructure-ui#1395]
### Background Information

**Package Version(s)**:

InstUI v8.49.0

**Browser**:

All Versions

**OS:**

MacOS Ventura (13.5)

**Device:**

Macbook Pro Max

**Component**:

Option.Item

### Describe the Issue

Ticket describing issue: https://instructure.atlassian.net/browse/OUT-5889

Quick Description of Issue:

- When using the keyboard to navigate options in an Option component, the option items are not read aloud by ScreenReader
- To fix this, the Option.Item role can be changed (e.g. "role = button")
- When I change the role to become a button, the component creates two spans, the outer span has "role = none" and the inner span has "role = button"
- Since arrow navigation lands on the outer span with (role = "none"), the text from the option item is not read by ScreenReader

### Steps To Reproduce

1. Enable "Improved Outcomes Management" Feature Flag in Canvas
2. Go to any course and navigate to the Outcomes page
3. Create 3 Outcome Groups all under the root outcome group for the course
4. In the top right corner, select "+ Create"
5. Navigate to the bottom of the Modal and select the "Location" Option menu
6. Turn on "Voice Over" (Cmd + F5)
7. Use arrow keys to navigate through the options and notice that none of the options are read aloud

### Expected Behavior

When moving into an Option.Item, the text within the Item should be read by ScreenReader

### Screenshots

View ticket for screenshots

### Additional Information

**Current Workaround(s):**

No current workarounds possible

**Products Affected:**

Outcomes page within Canvas is affected

**Are you willing to submit a PR to fix?**
- [x] Yes, I'm willing to submit a PR

**Requested Priority:**

Technically Blocking, but not super high priority

**Comment by balzss** [comment]
Hi, thanks for reporting this issue, I'll take a look

**Comment by jason-instructure** [comment]
Hey there, has there been any movement on this? Just want to check in :)

**Comment by balzss** [comment]
hi Jason, sorry for the late follow up. If I understand you correctly, adding the `role` prop to the outer span would solve your issue, right?
